mix.exs

```diff
@@ -30,19 +31,19 @@ defmodule App.Mixfile do
   #
   # Type `mix help deps` for examples and options.
   defp deps do
-    [{:phoenix, "~> 1.2.1"},
+    [{:phoenix, "~> 1.2"},
      {:phoenix_pubsub, "~> 1.0"},
      {:phoenix_ecto, "~> 3.0"},
      {:postgrex, ">= 0.0.0"},
      {:phoenix_html, "~> 2.6"},
      {:phoenix_live_reload, "~> 1.0", only: :dev},
      {:gettext, "~> 0.11"},
      {:cowboy, "~> 1.0"},
-     {:absinthe, "~> 1.2.0"},
-     {:absinthe_plug, "~> 1.1"},
-     {:absinthe_ecto, git: "https://github.com/absinthe-graphql/absinthe_ecto.git"},
```

Comment:
As the `mix.lock` determines package versions, nothing will be updated until a `mix deps.update`, so things should be fine. Still, most things here are fairly fast moving, so breakages are likely, I guess, from time to time.

I'm not clear which package you're referring to, tbh, though.

Regardless, it'd be nice to get some more tests around all of this to lessen the chances of something falling between the cracks.